packages/@angular/cli/models/webpack-configs/styles.ts

```diff
@@ -57,10 +57,10 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
           }
 
           if (deployUrl.match(/:\/\//)) {
-            // If deployUrl contains a scheme, ignore baseHref use deployUrl as is.
+            // If deployUrl contains a scheme, ignore baseHref and use deployUrl as is.
             return `${deployUrl.replace(/\/$/, '')}${URL}`;
           } else if (baseHref.match(/:\/\//)) {
-            // If baseHref contains a scheme, include it as is.
+            // If baseHref contains no scheme, include it as is.
             return baseHref.replace(/\/$/, '') +
```

Comment:
The original comment is correct.  The condition checks for the presence of '://' which would indicate the usage of a scheme.